typescript/sdk/src/index.ts

```diff
@@ -27,10 +27,10 @@ export {
   testSealevelChain,
 } from './consts/testChains.js';
 export {
+  attachAndConnectContracts,
```

Comment:
Still not sure why different devs get diff results for the auto-organize there but mine looks correct (it's alphabetically sorted). Feel free to ignore changes here, just added a few exports.